pkg/controller/utils/clusterinfo.go

```diff
@@ -100,6 +100,21 @@ func GetClusterRegion(kclient client.Client) (string, error) {
 	return infra.Status.PlatformStatus.AWS.Region, nil
 }
 
+// GetMasterNodes returns a machineList object whose .Items can be iterated
+// over to perform actions on/with information from each master machine object
+func GetMasterNodes(kclient client.Client) (*machineapi.MachineList, error) {
```

Comment:
I think this should be GetMasterMachineList

Nodes are a different thing in Openshift than machines.